backend/admin/service.go

```diff
@@ -103,39 +103,9 @@ func NewAdminService(
 	}
 }
 
-func Start(
-	ctx context.Context,
-	config Config,
-	cm *manager.Manager[configuration.Configuration],
-	sm *manager.Manager[configuration.Secrets],
-	schr ftlv1connect.SchemaServiceClient,
-	source *schemaeventsource.EventSource,
-	timelineClient *timelineclient.Client,
-	storage *artefacts.OCIArtefactService,
-	waitFor []string) error {
-
-	logger := log.FromContext(ctx).Scope("admin")
-
-	svc := NewAdminService(
-		config,
-		&EnvironmentManager{schr: NewSchemaRetriever(source), cm: cm, sm: sm},
-		schr,
-		source,
-		storage,
-		routing.NewVerbRouter(ctx, source, timelineClient),
-		timelineClient,
-		waitFor,
-	)
-
-	logger.Debugf("Admin service listening on: %s", config.Bind)
-	err := rpc.Serve(ctx, config.Bind,
-		rpc.GRPC(adminpbconnect.NewAdminServiceHandler, svc),
-		rpc.GRPC(ftlv1connect.NewVerbServiceHandler, svc),
-	)
-	if err != nil {
-		return fmt.Errorf("admin service stopped serving: %w", err)
-	}
-	return nil
+func (s *Service) Services(context.Context) ([]rpc.Option, error) {
```

Comment:
Maybe we should call this `StartService()` or `ConfigureService()`? eg. with the console it is also doing other stuff